warehouse/templates/manage/release.html

```diff
@@ -121,6 +121,30 @@ <h3>{% trans %}No files found{% endtrans %}</h3>
 
   <h3>{% trans %}Release settings{% endtrans %}</h3>
 
+  <div class="callout-block callout-block">
+    <h3>{% trans %}Yank release{% endtrans %}</h3>
+    <p>
+    {% if files %}
+      {% trans count=files|length %}
+        Yanking will mark this release (and {{ count }} file within it) to be ignored when installing in most common scenarios.
+      {% pluralize %}
+        Yanking will mark this release (and {{ count }} files within it) to be ignored when installing in most common scenarios.
+      {% endtrans %}
+    {% else %}
+      {% trans %}
+        Yanking will mark this release to be ignored when installing in most common scenarios.
+      {% endtrans %}
+    {% endif %}
+    {% trans project_name=project.name, version=release.version %}
+      This release will still be installable for users pinning to this exact version, e.g. when using <code>{{ project_name }}=={{ version }}</code>.
+    {% endtrans %}
+    {% trans href="https://www.python.org/dev/peps/pep-0592/" %}
+      For more information, see <a href="{{ href }}">PEP 592</a>.
```

Comment:
This doesn't need to block this PR, but I've wondered if our propensity for linking to PEPs is the best user experience we can provide here, or if we should figure out a real documentation pipeline for Warehouse (not really counting what exists now since it's primarily developer facing documentation, not user facing) and start moving to that.

Reply:
I think I agree, we are sort of hitting the limits of what we can document inline and in /help.

Reply:
I filed #7817 - thanks for the idea, @dstufft